docs/constants/mocha_versions.js

```diff
@@ -1,9 +1,9 @@
 const mochaVersions = Object.freeze({
-  "app-latest-tag": "v1.0.0-rc14",
-  "app-latest-sha": "cdc6bff4cde817a0f9fd6b30b57a02ad44698d17",
-  "core-latest-tag": "v1.24.0-tm-v0.34.28",
-  "core-latest-sha": "d24c81afe0ecc4ccfbb7eb543a45a11794f04db4",
-  "node-latest-tag": "v0.11.0-rc12",
-  "node-latest-sha": "d803cf847cfac3640b1272b3accc7f5940068888",
+  "app-latest-tag": "v1.0.0-rc15",
```

Comment:
Confirming https://github.com/celestiaorg/celestia-app/releases/tag/v1.0.0-rc15 doesn't introduce any breaking changes from https://github.com/celestiaorg/celestia-app/releases/tag/v1.0.0-rc14 so mocha should be able to bump to rc15 no problem. I'm currently running a rc-15 node on mocha-4 and it's syncting to tip w/ no issues.